packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/nodejs/configorder.md

```diff
@@ -1,13 +1,13 @@
 When multiple çonfigurations options are used simultaneously, the SDK chooses
 the option based on the following order:
 
-* SDK Client constructor
+* SDK `config.js` file
```

Comment:
The sample app uses configs from `config.js`, and the `config.js` loads information from environment variables (process.env). Devs can add env vars by either exporting from shell or using the `testenv` file.

Reply:
Thanks, added this info.